MAINTAINER.md

```diff
@@ -12,6 +12,46 @@
 - Julien Salleyron [@juliens](https://github.com/juliens)
 - Nicolas Mengin [@nmengin](https://github.com/nmengin)
 
+
+## PR review process:
+
+* If needed, a specific maintainer can be required to make a review (according to its knowledge areas)
+* The status `needs-design-review` is only used in complex/heavy/tricky PRs.
+* From `1` to `2`: 1 design LGTM in comment, by a senior maintainer, if needed.
+* From `2` to `3`: 3 LGTM by any maintainer.
+
+We use [PRM](https://github.com/ldez/prm) to manage locally pull requests.
+
+
+## Bots
+
+### [Myrmica Lobicornis](https://github.com/containous/lobicornis/)
+
+**Update and Merge Pull Request**
+
+As maintainer when you add the third LGTM, you must add `status/3-needs-merge`.
+If you want to preserve commits you must add `bot/merge-method-rebase` before `status/3-needs-merge`.
```

Comment:
Shall we add a note about the exceptions as to when vendor changes are involved? AFAICS, we do it as described followingly:

_If a PR involves changes to third-party dependencies, the commits pertaining to the `vendor` folder and the manifest/lock file(s) should be committed separated, respectively. In such cases, the rebase merge method should be selected and any squash-worthy commits be squashed manually by the author(s)._